src/lib/matchStatistics.ts

```diff
@@ -428,18 +438,75 @@ class MatchStatistics {
     };
   }
 
-  static whichColorWinsTheMost(matches: Match[]): {
-    color: "White" | "Black";
-    winPercentage: number;
-  } {
+  // static mostGamesInOneDayByPlayer(matches: Match[]) {
+  //   const matchesPerDate = matches.reduce(
+  //     (acc, curr) => {
+  //       const date = new Date(
+  //         curr.createdAt.getFullYear(),
+  //         curr.createdAt.getMonth(),
+  //         curr.createdAt.getDate(),
+  //       ).getTime();
+  //       if (!acc[date]) {
+  //         acc[date] = {
+  //           [curr.whitePlayerOne]: 1,
+  //           [curr.whitePlayerTwo]: 1,
+  //           [curr.blackPlayerOne]: 1,
+  //           [curr.blackPlayerTwo]: 1,
+  //         };
+  //       } else {
+  //         acc[date][curr.whitePlayerOne] += 1;
+  //         acc[date][curr.whitePlayerTwo] += 1;
+  //         acc[date][curr.blackPlayerOne] += 1;
+  //         acc[date][curr.blackPlayerTwo] += 1;
+  //       }
+  //       return acc;
+  //     },
+  //     {} as Record<number, Record<string, number>>,
+  //   );
+
+  //   const mostGamesOnOneDay = Math.max(
+  //     ...Object.values(matchesPerDate).map((date) =>
+  //       Math.max(...Object.values(date)),
+  //     ),
+  //   );
+
+  //   const dayWithMostGames = Number(
+  //     Object.keys(matchesPerDate).find((key) => {
+  //       const date = matchesPerDate[Number(key)];
+  //       return Math.max(...Object.values(date)) === mostGamesOnOneDay;
+  //     }),
+  //   );
+
+  //   const playerWithMostGames = Object.keys(
+  //     matchesPerDate[dayWithMostGames],
+  //   ).find(
+  //     (key) => matchesPerDate[dayWithMostGames][key] === mostGamesOnOneDay,
+  //   );
+
+  //   return {
+  //     date: new Date(dayWithMostGames),
+  //     player: playerWithMostGames,
+  //     games: mostGamesOnOneDay,
+  //   };
+  // }
+
+  static winsByResult(matches: Match[]) {
     const whiteWins = matches.filter((mt) => mt.result === "White").length;
     const blackWins = matches.filter((mt) => mt.result === "Black").length;
-    const totalGames = matches.filter((x) => x.result !== "Draw").length;
+    const totalGames = matches.length;
+    const numOfDraws = matches.filter((mt) => mt.result === "Draw").length;
     const whiteWinPercentage = (whiteWins / totalGames) * 100;
     const blackWinPercentage = (blackWins / totalGames) * 100;
-    return whiteWinPercentage > blackWinPercentage
-      ? { color: "White", winPercentage: whiteWinPercentage }
-      : { color: "Black", winPercentage: blackWinPercentage };
+
+    return {
+      blackWins: { wins: blackWins, procentage: blackWinPercentage },
+      whiteWins: { wins: whiteWins, procentage: whiteWinPercentage },
+      totalGames: totalGames,
+      numOfDraws: {
+        draws: numOfDraws,
+        procentage: (numOfDraws / totalGames) * 100,
```

Comment:
rename to `percentage` :smile: 
profile.tsx also breaks from this since it used the old method.